builders/integrations/indexers/covalent.md

```diff
@@ -127,39 +142,30 @@ Copy and paste the below code block into your preferred environment, or [JSFiddl
 	}
 
 	getData();
-
     ```
 
-The output should resemble the below. The balances endpoint returns a list of all ERC-20 and NFT token balances including ERC-721 and ERC-1155 along with their current spot prices (if available).
+The balances endpoint returns a list of all ERC-20 and NFT token balances, including ERC-721 and ERC-1155 balances, along with their current spot prices (if available).
 
 ![Javascript Console Output](/images/builders/integrations/indexers/covalent/covalentjs.png)
 
 ## Python Example {: #python-example } 
+
 Covalent doesn’t have an official API wrapper. To query the API directly you will have to use the Python [requests library](https://pypi.org/project/requests/). Install requests into your environment from the command line with `pip install requests`. Then import it and use it in your code. Use the HTTP verbs get methods to return the information from the API. Copy and paste the below code block into your preferred environment and run it. The output should look similiar to the screenshot above, however the formatting may vary depending on your environment.
 
 ```python
 import requests
 
 def fetch_wallet_balance(address):
 	api_url = 'https://api.covalenthq.com'
-    endpoint = f'/v1/1287/address/{address}/balances_v2/'
+    endpoint = f'/v1/{{ networks.moonbase.chain_id }}/address/{address}/balances_v2/'
     url = api_url + endpoint
-    r = requests.get(url, auth=('YOUR API KEY HERE',''))
-    print(r.json())
-    return(r.json())
+    response = requests.get(url, auth=('YOUR API KEY HERE',''))
+    print(response.json())
+    return(response.json())
 
-#Example address request
+# Example address request
 fetch_wallet_balance('0xFEC4f9D5B322Aa834056E85946A32c35A3f5aDD8')
-
 ```
 
 !!! note
     The 2nd parameter of `auth` is empty, because there is no password required - your API key is all that's needed.
-
-### Community Built Libraries {: #community-built-libraries } 
-Covalent currently has libraries in Python, Node, and Go, which are built and maintained by the community as part of the [Covalent Alchemists Program](https://www.covalenthq.com/ambassador/). The tools have been built by the community to provide value to users of the Covalent API and are [available here](https://www.covalenthq.com/docs/tools/community).
```

Comment:
Why was this removed?

Reply:
the link to the community tools page gives back a 404 and I could not find a replacement page that talks about the Python, Node, and Go libraries anywhere in the official Covalent docs. So I didn't think we should mention it if they aren't mentioning it. Then that left the Alchemists Program link by itself and that was kind of weird to me... I figured it was better to remove the whole section